README.md

```diff
@@ -163,6 +163,28 @@ HTML:
 </table></d2l-table-wrapper>
 ```
 
+#### Lightweight Style - Full Design Spec Still In Progress
+
+<img src="/images/lightweight.png?raw=true" width="350">
+
+Lightweight data tables are used to display tabular data in rows when column dividers or individual cell colors are not required. They are best suited for data that is primarily organized by row while still allowing users to order the rows by using column header sorting.
+
+HTML:
+```html
+<!-- use the style include in custom components -->
+<style include="d2l-table-lightweight-style"></style>
```

Comment:
How would this approach of including the lightweight style work on pages where you wanted both a lightweight and a "normal" table?

Reply:
Hmm, TBH I hadn't considered us wanting to do that, but it could be possible for a complicated page.  Having to wrap your table in a component doesn't feel great, but I'm not seeing an obvious solution...

Reply:
I could also add an attribute to the table level, and all the overriding styles are only applied if that's present

Reply:
Yeah that's what I was thinking initially before I saw the code... something like `<d2l-table type="default|light">`? Our legacy grids [had exactly this concept](https://docs.dev.d2l/index.php/Grid_Control) and could be put into "data" or "list" mode.